### PR TITLE
styles

### PR DESCRIPTION
**Pull request by fobabett**

### CODE REVIEW

File: ui/src/components/Hash.vue [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-      <h3>{{hashedName}}</h3>
-    </div>
-    <div class="errors">
-      <p>{{errors}}</p>
+    <h2>Or enter some key <span class="show-for-large">to create a hashed random phrase</span></h2>
+    <div class="inner">
+      <div>
+        <input type="text" placeholder="your keywords (ex. v1.0.3)" v-on:keyup="generate" v-model="key" />
+      </div>
+      <input class="output" placeholder="result" value="key" disabled/>
```
Comment (theRemix) [inline ui/src/components/Hash.vue]: this should display the value of `{{hashedName}}`

**Review by theRemix — CHANGES_REQUESTED** [review]

File: ui/src/components/Hash.vue (hunk 1 above)
Comment (theRemix) [reply ui/src/components/Hash.vue]:
```html
      <input class="output" placeholder="result" v-model="hashedName" disabled/>
```
should do the trick

File: ui/src/components/ApiTabPanel/content/ContentCurl.vue [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-    <code><pre>curl 90s.fun/random</pre></code>
-    <code><pre>curl 90s.fun/names/v1.2.0-rc8</pre></code>
-    <code><pre>curl 90s.fun/names/2017-06-14T05:10:48.935Z</pre></code>
+  <div class="content-curl code">
+    <span class="code-syntax-pink">curl 90s.fun/random</span>
+    <br>
+    <span class="code-syntax-blue">curl 90s.fun/names/v1.2.0-rc8</span>
+    <br>
+    <span>curl 90s.fun/names/2017-06-14T05:10:48.935Z</span>
   </div>
```
Comment (theRemix) [inline ui/src/components/ApiTabPanel/content/ContentCurl.vue]: add "See example on Github" outside of the code block

File: ui/src/components/ApiTabPanel/content/ContentGo.vue [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
+    <span class="code-syntax-blue">func </span><span class="code-syntax-pink">main </span><span>() {</span>
+    <div class="code-inner">
+      <span>fmt.Println(<span class="code-syntax-blue">go90s.GetRandomName></span>())</span>
+      <br>
 
+      <span>fmt.Println(<span class="code-syntax-blue">go90s.GetName></span>(<span class="code-syntax-pink">v1.2.0-rc8"</span>))</span>
+    </div>
+    <span>}</span>
   </div>
+  <i>See: <a href="https://github.com/theRemix/Nineties-Namer/blob/master/routes/router.go">Example on Github</a></i>
```
Comment (theRemix) [inline ui/src/components/ApiTabPanel/content/ContentGo.vue]: move the "See example on Github" to outside of the code block

File: ui/src/components/ApiTabPanel/content/ContentJs.vue [hunk 4]
```diff
@@ -1,21 +1,33 @@
...
+        <span>.<span class="code-syntax-blue">then </span>(<span class="code-syntax-pink"> res </span>=> <span class="code-syntax-pink">res</span>.text() )</span>
+        <br>
+        <span>.<span class="code-syntax-blue">then </span>(<span class="code-syntax-pink"> randomName</span> => {</span>
+        <div class="code-inner">
+          <span><span class="code-syntax-blue">console</span>.<span class="code-syntax-blue">log</span>( <span class="code-syntax-pink">randomName  </span>);</span>
+        </div>
+        <span>})</span>
+      </div>
+    </div>
     <i>See: <a href="https://github.com/theRemix/Nineties-Namer/blob/master/ui/src/components/Random.vue">Random Example on Github</a></i>
```
Comment (theRemix) [inline ui/src/components/ApiTabPanel/content/ContentJs.vue]: move the "See example on Github" to outside of the code block

File: ui/src/components/ApiTabPanel/content/ContentJs.vue [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
+        <br>
+        <span>.<span class="code-syntax-blue">then</span>( <span class="code-syntax-pink">generatedName</span> => {</span>
+        <div class="code-inner">
+          <span><span class="code-syntax-blue">console</span>.<span class="code-syntax-blue">log</span>(<span class="code-syntax-pink"> generatedName</span> );</span>
+        </div>
+        <span>})</span>
+      </div>
+    </div>
 
     <i>See: <a href="https://github.com/theRemix/Nineties-Namer/blob/master/ui/src/components/Hash.vue">Names Example on Github</a></i>
```
Comment (theRemix) [inline ui/src/components/ApiTabPanel/content/ContentJs.vue]: move the "See example on Github" to outside of the code block

**Review by theRemix — CHANGES_REQUESTED** [review]
the "See example on Github" links should be outside of the dark gray code blocks

**Review by theRemix — CHANGES_REQUESTED** [review]
add google fonts please
https://projects.invisionapp.com/share/43C7EQ1F8#/screens/239931045

**Comment by fobabett** [comment]
made requested changes

**Review by theRemix — APPROVED** [review]

**Pull request merged**